src/main/java/uk/gov/justice/hmpps/casenotes/services/CaseNoteService.java

```diff
@@ -167,6 +169,7 @@ private CaseNote mapper(final NomisCaseNote cn, final String offenderIdentifier)
                 .occurrenceDateTime(cn.getOccurrenceDateTime())
                 .authorName(cn.getAuthorName())
                 .authorUsername(String.valueOf(cn.getStaffId()))  // TODO: this should be username not id.
+                .authorUserId(String.valueOf(cn.getStaffId()))
```

Comment:
I'll remove `authorUsername` in prisonstaffhub before removing it from here

Reply:
Ok